src/wdio/services/TerraCommands/accessibility.js

```diff
@@ -10,8 +10,11 @@ import determineOptions from './determine-test-options';
  * @param {Object[]} [options.viewports] - the list of Terra viewports to test.
  */
 const runAccessibilityTest = (options) => {
-  const axeResults = global.browser.axe(options);
-  global.expect(axeResults).to.be.accessible();
+  const { theme } = global.browser.options;
+  if (theme !== 'clinical-lowlight-theme') {
```

Comment:
Is this for short-term or long-term? In either case, I think it is better to have a follow-up issue to remove this check whenever a decision is made to test accessibility in lowlight.